addons/game.controller.genesis.teamplayer/addon.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="game.controller.genesis.teamplayer"
+        name="Genesis Team Player"
+        version="1.0.0"
+        provider-name="Team Kodi">
+    <extension point="kodi.game.controller" library="resources/layout.xml"/>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <assets>
+            <icon>resources/icon.png</icon>
+        </assets>
+        <summary lang="en_GB">Genesis Team Player</summary>
+        <description lang="en_GB">Team Player is a peripheral released by Sega for the Sega Genesis console, allowing for up to four players to plug Genesis controllers into the unit.</description>
```

Comment:
I try to include release year (1988) in the description, because it lets readers calibrate the historical significance of the work you've done.

We can also aim for more concise sentences:

> Team Player is a peripheral released by Sega for the Sega Genesis in 1988. It allows up to four additional players to connect Genesis controllers.

Reply:
I've changed the description.